bosh/ruby-app-release/jobs/rubyweb/spec

```diff
@@ -0,0 +1,14 @@
+---
+name: rubyweb
+
+templates:
+  ctl.ebr: tmp/ctl
+  config.ebr: tmp/config.yml
+
+packages:
+- ruby
+
+properties:
+  port:
+    description: "Specify the port for the Web-app to listen for"
```

Comment:
```suggestion
    description: "Specify the port for the Web-app to listen on"
```

Reply:
Semantically speaking, `on` is the correct term. The app listens _for_ requests _on_ a port.

But I will simplify the sentence cause it is a bit [Americanese](https://www.urbandictionary.com/define.php?term=Americanese).